sdk/inc/azure/iot/az_iot_hub_client_properties.h

```diff
@@ -28,11 +28,9 @@
  * The payload will be of the form:
  *
  * @code
- * "reported": {
- *     "<component_name>": {
- *         "__t": "c",
- *         "temperature": 23
- *     }
+ * "<component_name>": {
```

Comment:
This was just a doc comment bug fix, right, not an actual change to the payload format being sent over the wire to/from the service?

Reply:
Yup only documentation